README.adoc

```diff
@@ -12,7 +12,7 @@ toc::[]
 https://github.com/openshift/odo/releases/latest[image:https://img.shields.io/github/v/release/openshift/odo?style=for-the-badge[GitHub release]]
 https://travis-ci.com/openshift/odo[image:https://img.shields.io/travis/com/openshift/odo?style=for-the-badge[Build Status]]
 image:https://img.shields.io/github/license/openshift/odo?style=for-the-badge[License]
-
+https://coreos.slack.com/messages/CHS495NE9[image:https://img.shields.io/badge/Slack-Join%20Chat-yellowgreen?style=for-the-badge&logo=slack[Slack]]
```

Comment:
Doesn't this need a Red Hat account?

Reply:
Ah I think you're right. The CoreOS slack isn't for everyone.